polaris-react/src/components/Autocomplete/components/MappedOption/MappedOption.scss

```diff
@@ -13,7 +13,6 @@
 
   svg {
     fill: var(--p-color-icon);
-    color: var(--p-color-icon-on-color);
```

Comment:
These aren't doing anything, the icons are already on a white background (the autocomplete popover surface)